src/corelocation.cs

```diff
@@ -407,6 +423,47 @@ partial interface CLLocationManager {
 		[Mac (10,14)]
 		[Export ("requestLocation")]
 		void RequestLocation ();
+
+		[Watch (7, 0), TV (14, 0), Mac (10, 16), iOS (14, 0)]
+		[Export ("accuracyAuthorization")]
+		CLAccuracyAuthorization AccuracyAuthorization { get; }
+
+		// TODO: Apple??? 
```

Comment:
??? please add this to the main tracking issue
or we're likely to forget it
In general we should not expose new/deprecated API

Reply:
This is super weird...the impression I got from WWDC is that all of these location authorization APIs were one of the big new features 🤔

Reply:
ditto, I bet it was a screw up in the headers.